Makefile

```diff
@@ -22,6 +23,10 @@ clean-pyc:
 	find . -name '*.pyo' -exec rm -f {} +
 	find . -name '*~' -exec rm -f {} +
 
+format: 
+	python3 -m black .
```

Comment:
One last thought. It's common to put a virtual environment folder `venv/` at the root of the repository, which contains a lot of internal python files. Is it possible to ignore these folders, since they may contain thousands of python files?

https://github.com/mozilla/probe-scraper/blob/022eb2d102ffcdd726e012dfd3d2a2dc2374b776/.gitignore#L81-L83

If you wanted to test this, run `python3 -m venv; source venv/bin/activate;' pip install -r requirements.txt`, then run `make format` and check what's getting formatted.